src/components/PopUpBar/PopUpBar.css

```diff
@@ -1,10 +1,12 @@
 .popup_container {
-  position: flex;
+  position: fixed;
```

Comment:
I have suggested Position sticky is not fixed. With sticky users will get a little better experience
```
position: sticky
top:0
```

It's very small change  do it and I will approve this one 
thanks